### DISCUSSION TITLE
Discharge capacity seems erroneous

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3685] [Q&A] [answered]
Hello, 

I tried to obtain discharge capacity using the attached code, But it shows a reverse trend when I plot it, as seen in the figure below. 

Can someone help with this, I have also noticed a similar previous discussion that does not contain an answer for it. 

![image](https://github.com/pybamm-team/PyBaMM/assets/125870976/42d743f1-30e0-4913-92e0-b7467d79b6c9)

[OCV and Discharge capacity.pdf](https://github.com/pybamm-team/PyBaMM/files/13824091/OCV.and.Discharge.capacity.pdf)

**Comment by valentinsulzer** [comment, marked as answer]
Discharge capacity always starts at zero then increases during a discharge. The convention we use is that current is positive for a discharge and negative for a charge.

**Reply by Akila1993** [reply]
@tinosulzer Thank you for your clarification.
